docs/how-to-write-conventions/README.md

```diff
@@ -183,6 +187,27 @@ to that system. For example, when adding a new `db.system.name` value, create
 documentation and span definitions that show how generic attributes work for that
 database.
 
+### Defining entities
```

Comment:
Why are Entities no longer signals?

Let's not refactor the layout beyond adding which signal to use.

See my comment about state/config + entity.